flink-test-utils-parent/flink-test-utils/src/main/java/org/apache/flink/test/util/AbstractTestBase.java

```diff
@@ -70,6 +70,11 @@
 
     @After
     public final void cleanupRunningJobs() throws Exception {
+        if (!miniClusterResource.getMiniCluster().isRunning()) {
+            // do nothing if the MiniCluster is not running
```

Comment:
NIT: Is it a normal situation? If not, maybe it makes sense to log a warning?

Reply:
It is not a normal situation. I'll add a LOG there.